core/contracts/financial-templates/implementation/Liquidatable.sol

```diff
@@ -189,16 +189,22 @@ contract Liquidatable is PricelessPositionManager {
         address sponsor,
         FixedPoint.Unsigned calldata collateralPerToken,
         FixedPoint.Unsigned calldata maxTokensToLiquidate
-    ) external fees() onlyPreExpiration() returns (uint liquidationId) {
+    )
+        external
+        fees()
+        onlyPreExpiration()
+        returns (
+            uint liquidationId,
+            FixedPoint.Unsigned memory tokensLiquidated,
+            FixedPoint.Unsigned memory finalFeeBond
+        )
+    {
         // Retrieve Position data for sponsor
         PositionData storage positionToLiquidate = _getPositionData(sponsor);
 
-        FixedPoint.Unsigned memory tokensToLiquidate = FixedPoint.min(
-            maxTokensToLiquidate,
-            positionToLiquidate.tokensOutstanding
-        );
+        tokensLiquidated = FixedPoint.min(maxTokensToLiquidate, positionToLiquidate.tokensOutstanding);
```

Comment:
Because of this, which could make `maxTokensToLiquidate !== tokensLiquidated`, this return value is very useful.